Products/DataCollector/plugins/zenoss/snmp/InterfaceMap.py

```diff
@@ -156,7 +156,7 @@ def process(self, device, results, log):
 
                 elif len(ip_parts) == 16:
                     addr_type = IpUtil.IPV6_ADDR_TYPE
-                    ip = IpUtil.bytesToCanonIp(ip_parts)
+                    ip = IpUtil.bytesToCanonIp(ip_parts) + '/64'
```

Comment:
Appears that we're just picking a frequently used subnet and assuming that it is the correct subnet. We should fetch the IPv6 subnet using the IPv6-MIB. This will likely require some significant changes to the InterfaceMap plugin.